common/app/views/fragments/footer.scala.html

```diff
@@ -181,3 +166,24 @@
         </a>
     </div>
 }
+
+@emailSignUpLink(currentEdition: Edition) = {
+    <section class="footer__email-container">
+        <div class="colophon__section">
+        @currentEdition match {
+            case Uk => {
+                @fragments.email.signup.emailFooterLink("morning-briefing", "/global/ng-interactive/2022/apr/13/first-edition-sign-up-guardian")
```

Comment:
`signupPage` is an attribute of newsletters retrieved from the Newsletters API. Do you think we could use that attribute here?